pipeline.yml

```diff
@@ -0,0 +1,47 @@
+version: 3.0.1
+pipeline_os: windows
+create_github_release: true
+create_pr_release_to_master: true
+
+env:
+  - GITHUB_ACCESS_TOKEN_ID: github_access_token_acsbuildguy
+  - JENKINS_NODE_WIN: DynamoWinBuild
+
+code_analysis:
+  sonarqube:
+    source_encoding: UTF-8
+    src: .
+    coverage_reports:
+      - coverage.info
+
+build:
+  -
+    scripts:
+      - dotnet msbuild src/Autodesk.RefineryToolkits.sln -p:Configuration=Release -p:CopyLocal=False -restore
+
+deployment:
+  -
+    type: sign
+    files_to_sign:
+      - build/bin/Autodesk.RefineryToolkits.*.dll
+  -
+    type: customized
+    scripts:
+      - "pwsh.exe -ExecutionPolicy ByPass -Command Compress-Archive -Path build -DestinationPath RefineryToolkit.zip"
+  -
+    type: artifacts
+    publish_to_jenkins: true
+    publish_to_artifactory: false
+    allow_branches: ".*"
+    outputs:
+      - RefineryToolkit.zip
+  -
+    type: artifacts
+    publish_to_jenkins: false
+    publish_to_artifactory: true
+    outputs:
+      - RefineryToolkit.zip
```

Comment:
Can we not just say:
```
type: artifacts
    publish_to_jenkins: true
    publish_to_artifactory: true
    allow_branches: ".*"
    outputs:
      - RefineryToolkit.zip
      
```
      instead of repeating this block?

Reply:
Then we'd get an artifactory package on pr builds, not sure if we want that

Reply:
> Then we'd get an artifactory package on pr builds, not sure if we want that

Yes, sorry, I meant without the `allow_branches: ".*"`

Reply:
Ah, right. That was when I thought I could run this on each commit to the cicd branch. Very well might end up there if I need a faster debugging loop, but yes, since nothing outside of master gets seen by Jenkins, we can simplify.